docs/releases/breaking-changes.md

```diff
@@ -13,6 +13,17 @@ these to a minimum, but rest assured we only make them with good reason.
 
 Starting with Directus 10.0, here is a list of potential breaking changes with remedial action you may need to take.
 
+## Version [NEXT]
+
+### Introduced Allow List for OAuth/OpenID Redirects
+
+Due to an Open Redirect security issue with the OAuth and OpenID SSO providers we have introduced an allow list for
+these redirects. If your current workflow depends on redirecting to an external domain after successful SSO login using
+the `?redirect=http://example.com/login` query parameter then you'll need to add this URL to the
+`AUTH_<PROVIDER>_REDIRECT_ALLOW_LIST`.
+
+# `AUTH_<PROVIDER>_REDIRECT_ALLOW_LIST` accepts a comma separated list of URLs (which include the path when comparing).
```

Comment:
Not 100% sure, but maybe this wasn't meant to be a h1 tag, perhaps as an info or tip block instead?